hamilton/plugins/polars_extensions.py

```diff
@@ -108,7 +111,7 @@ class PolarsCSVReader(DataLoader):
 
     @classmethod
     def applicable_types(cls) -> Collection[Type]:
-        return [DATAFRAME_TYPE]
+        return [DATAFRAME_TYPE, pl.LazyFrame]
```

Comment:
We don't want this here -- only for Loaders.
```suggestion
        return [DATAFRAME_TYPE]
```

Reply:
sorry I meant savers not loaders.